doc/api/stream.md

```diff
@@ -572,6 +572,12 @@ Examples of writable streams include:
 * [TCP sockets][]
 * [child process stdin][]
 * [`process.stdout`][], [`process.stderr`][]
+ 
+#### Event: 'close'
+
+Emitted when the stream and any of its underlying resources (a file descriptor, for example) have been closed. The event indicates that no more events will be emitted, and no further computation will occur.
```

Comment:
How does this differs from `'end'`? 
Should a user expect `'close'` or `'end'` to proceed? Which one happens first?

I will also move this block after `'end'`, meaning that `'end'` is the preferred way of waiting for the end of a stream.